.github/workflows/gem.yml

```diff
@@ -20,3 +27,15 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby_version }}
           bundler-cache: true
+      - name: Extract Plugin Version
+        id: plugin-meta
+        run: echo "::set-output name=version::$(bundle exec ruby -e 'require "jekyll_test_plugin"; puts JekyllTestPlugin::VERSION')"
+        shell: bash
+      - name: "Build Gem (JekyllTestPlugin ${{ steps.plugin-meta.outputs.version }})"
+        run: gem build jekyll_test_plugin.gemspec
```

Comment:
Eventually `gem push` would be great to add here too since most folks download from RubyGems.org!

Reply:
Yes. That's the whole point of using this repository for testing.
The third stage of testing involves checking whether the API Key is accessible via this repo.. 
(*The second stage being centered around tags*).